tests/testproject/TestProject.IntegrationServiceA/Program.cs

```diff
@@ -8,13 +8,15 @@
 builder.AddNpgsqlDataSource("postgresdb");
 builder.AddRabbitMQ("rabbitmqcontainer");
 builder.AddMongoDBClient("mymongodb");
+builder.AddOracleClient("freepdb1");
```

Comment:
This test program should be using the Oracle EF Component you are adding, not a workaround method and talking to an OracleConnection directly.

Reply:
Done. It makes perfect sense, I got a little lost here. Thank you for the explanation.